app/controllers/sessions_controller.rb

```diff
@@ -1,12 +1,10 @@
 class SessionsController < ApplicationController
-
   def new
-
   end
 
   def create
     @user = User.find_by_email(params[:session][:email])
-    if @user && @user.authenticate(params[:session][:password])
+    if @user&.authenticate(params[:session][:password])
```

Comment:
Is this valid ruby syntax?

Reply:
It would appear so!

> Style/SafeNavigation: Use safe navigation (&.) instead of checking if an object exists before calling the method.
    if @user && @user.authenticate(params[:session][:password])

I suppose it could be harder to read this way though?

Reply:
https://www.rubydoc.info/gems/rubocop/0.43.0/RuboCop/Cop/Style/SafeNavigation

Reply:
Ok  learned something new today then